endpoints/app.endpoints.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { customError } = require("../error-handling");
+const { getTopics } = require("../controllers/topics.controller");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (request, response, next) => {
+  const error = { status: 404, message: "Not a valid route" };
+  customError(error, request, response, next);
+});
```

Comment:
So this one isn't actually a custom error as you are not forcing the error to happen. Therefore, you can send your response directly in the callback without invoking `customError`. You can then use `customError` for when that is needed in a future ticket 😄